Cargo.toml

```diff
@@ -44,7 +44,8 @@ build-flac = ["cmake"]
 build-ogg = ["build-flac"]
 
 [build-dependencies]
-cmake = {version = "0.1", optional = true}
+# 0.1.45 adds cmake::Config::get_profile() which we need
```

Comment:
I think the comment is helpful, but "which we need" is redundant.
Because if we wouldn't need it, why mention it at all?